.ci/go-test.sh

```diff
@@ -11,8 +11,12 @@
 # "go test -timeout X"
 timeout_value=${KATA_GO_TEST_TIMEOUT:-10s}
 
+# -race flag is supported only on amd64/x86_64 arch , hence 
```

Comment:
Out of interest, why is this flag only supported on certain architectures? I wonder if there is an open bug on this we could reference here?

Reply:
I am not aware of the reason why it is not supported on the other arch expect x86_64, but the work is currently going on to support it on the other platforms as well. Refer the below open issue from the golang github

https://github.com/golang/go/issues/19273

Reply:
Thanks for the reference!